README.md

```diff
@@ -122,7 +122,7 @@ Now securely transfer the `meetingResponse` and `attendeeResponse` objects to yo
 These objects contain all the information needed for a client application using the Amazon Chime SDK for JavaScript to join the meeting.
 
 The value of the MediaRegion parameter in the createMeeting() should ideally be set to the one of the media regions which is closest to the user creating a meeting.
-An implementation can be found in the [getNearestMediaRegion()](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts#L460)
+An implementation can be found in the [Amazon Chime SDK media Regions documentation](https://docs.aws.amazon.com/chime/latest/dg/chime-sdk-meetings-regions.html)
```

Comment:
Request you to just update this to : An implementation can be found in the [Amazon Chime SDK Media Regions documentation](https://docs.aws.amazon.com/chime/latest/dg/chime-sdk-meetings-regions.html).

Adding a period (.) and capitalizing the media. I know the public documentation has it small cased.